bus/uploadedsectors.go

```diff
@@ -0,0 +1,60 @@
+package bus
+
+import (
+	"sync"
+	"time"
+
+	"go.sia.tech/core/types"
+	"go.sia.tech/renterd/api"
+)
+
+const cacheExpiry = 12 * time.Hour
```

Comment:
I considered making this 24h to cover (extremely) large uploads, since we mark uploads as finished anyway the pruning we do is there to avoid any leakage.

Reply:
Honestly why not. 24 hours is probably ok.